GrandCentralBoard/Widgets/Slack/SlackAvatarView.swift

```diff
@@ -12,7 +12,17 @@ import GCBCore
 
 final class SlackAvatarView: UIImageView {
 
-    override func layoutSubviews() {
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setUp()
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setUp()
+    }
+
+    private func setUp() {
         super.layoutSubviews()
```

Comment:
Why we call super.layoutSubviews() inside `init`?

Also from Apple documentation about `layoutSubviews`: 
> You should not call this method directly. If you want to force a layout update, call the setNeedsLayout method instead to do so prior to the next drawing update. If you want to update the layout of your views immediately, call the layoutIfNeeded method.